### ISSUE TITLE
Undefined variable $list in every model collection

### ISSUE THREAD

**Issue by filliph** [XeroAPI/xero-php-oauth2#117]
**SDK you're using (please complete the following information):**
1.6.3

**Describe the bug**
Every `jsonSerialize` method has a variable `$list` as the first argument to `AccountingObjectSerializer::sanitizeForSerialization` but this variable is undefined.

**To Reproduce**
1. Open every model collection like BankTransactions, Accounts, etc in an IDE with IntelliSense
2. Scroll down to `public function jsonSerialize()`
3. See error

**Expected behavior**
Judging from `public function __toString()` in those same files, `$list` should indeed be `$this` instead.

**Comment by SidneyAllen** [comment]
@filliph - can you share a snippet of the code which needs to change and the exact change required.  Happy to apply it to our templates so it propogates through all the models.

**Comment by filliph** [comment]
```php
    public function jsonSerialize()
    {
        return AccountingObjectSerializer::sanitizeForSerialization($list)->Accounts;
    }
```

Should be

```php
    public function jsonSerialize()
    {
        return AccountingObjectSerializer::sanitizeForSerialization($this)->Accounts;
    }
```

As `$list` is not defined.

**Comment by SidneyAllen** [comment]
Thanks @filliph 

I've applied the change and released version 1.6.4
